src/utils/locales/en.js

```diff
@@ -93,6 +93,28 @@ export default {
     tactics: 'Tactics',
   },
   animationEditor: {
+    untitledAnimation: 'Unamed animation',
```

Comment:
And is it animation or drill ? (We should choose a word and stick to it, the home page button says "new drill")

Reply:
I was wondering...  ok I will switch to drill.